plasmapy/formulary/parameters.py

```diff
@@ -1402,16 +1389,14 @@ def magnetic_energy_density(B: u.T):
     <Quantity 3978.8735773 J / m3>
 
     """
-    E_B = magnetic_pressure(B).to(u.J / u.m ** 3)
-    return E_B
+    return magnetic_pressure(B)
```

Comment:
Oh wow, I did not realize this was an alias all along... I wonder if there might not be a way to consolidate these two via some alias like

```python
magnetic_energy_density = validate_quantities({appropriate annotations})(magnetic_pressure)
```

and then appropriate docstring changes along the lines of "This function has two aliases, because..."... Obviously irrelevant to this PR, though!

Reply:
Yeah, I’ll create an issue for this. I feel the deciding point will hinder on how clean the docstrings are.